odoo/addons/base/models/assetsbundle.py

```diff
@@ -163,12 +174,17 @@ def to_node(self, css=True, js=True, debug=False, async_load=False, defer_load=F
                 if self.css_errors:
                     msg = '\n'.join(self.css_errors)
                     response.append(JavascriptAsset(self, inline=self.dialog_message(msg)).to_node())
+
             if js and self.javascripts:
+                if debug:
+                    url = self.js(is_minified=False)[0].url
+                else:
+                    url = self.js(is_minified=True)[0].url
```

Comment:
`self.js(is_minified=not debug)`?

Also a bit uncomfortable with the `is`, it reads like a predicate e.g. we're asking if it's minified, but if I understand correctly this is about telling the thing that we do / don't want the minified asset right?

Reply:
Yes is_minifed=True means that the minified assets will be returned. Minifed assets are only used in normal mode (Not debug).
Do you have a better name than is_minifed=True?

Reply:
> Do you have a better name than is_minifed=True?

Just `minified=True`. As in, you ask for the JS content, minified.